### PR TITLE
Create ContributionGuidelines.md

### LINKED ISSUE #25: CONTRIBUTIONGUIDELINES.md
The project could use a well illustrated contribution guidelines documentation. I would love to work on it.

### PR DESCRIPTION
**Pull request by kanishkaa24**
Closes #25

### CODE REVIEW

File: ContributionGuidelines.md [hunk 1]
```diff
@@ -0,0 +1,127 @@
+# Contributing Guidelines  
+  
+This documentation contains a set of guidelines to help you during the contribution process.
+
+We are happy to welcome all the contributions from anyone willing to improve/add new scripts to this project. Thank you for helping out and remember, **no contribution is too small.** 
+
+- Each participant/contributor will be assigned **2 issues (max)** at a time to work.
+- Participants/contributors have **7 days** to complete issues. After that issues will be assigned to others.
```
Comment (mishraaditya595) [inline ContributionGuidelines.md:8]: Change it to 3 days

Comment (kanishkaa24) [reply ContributionGuidelines.md:8]: Okay Definitely

File: ContributionGuidelines.md [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+- Pull Requests review criteria:
+  - Please fill the **PR template** properly while making a PR
+  - You must add your code file into the respective **folders**.
+  - Your work must be **original**, written by you not copied from other resources.
+  - You must **comment** on your code where necessary.
+  - For **frontend changes** kindly share screenshots and work samples of your work before sending a PR.
+  - Follow the proper [**style guides**](https://google.github.io/styleguide/) for your work.
+
+- For any queries or discussions, please drop a message in our **SWOC slack channel or DWOC discord server**
```
Comment (mishraaditya595) [inline ContributionGuidelines.md:28]: Remove it.

Comment (mishraaditya595) [reply ContributionGuidelines.md:28]: Make the template for all contributors. Not just for people from Open Source programs like DWOC.

Comment (kanishkaa24) [reply ContributionGuidelines.md:28]: Okay Sure

**Comment by kanishkaa24** [comment]
@mishraaditya595 
I have made the suggested changes.

**Comment by mishraaditya595** [comment]
Thanks for your contribution. A star to the repo is much appreciated.

**Comment by kanishkaa24** [comment]
Okay
This contribution is made for DWOC for Level 2. Please add the labels.
Thank you

**Pull request merged**